libraries/package-extractor/webpack.config.js

```diff
@@ -29,6 +29,10 @@ module.exports = () => {
       new webpack.ids.DeterministicModuleIdsPlugin({
         maxLength: 6
       })
+    ],
+    ignoreWarnings: [
```

Comment:
You can use `resolve.alias` and map it to `false` instead; then webpack will inject a stub that throws if it gets called, if memory serves

Reply:
Yeah that doesn't work here since other scenarios use graceful-fs